### DISCUSSION TITLE
HELP SD XL size mismatch for model

### DISCUSSION THREAD

**Discussion by juanman1000** [Mikubill/sd-webui-controlnet#1919] [Q&A]
Greetings friends, I have this problem, download the SD XL model, and update the automatic1111 - web ui, but when loading the model I get this error:
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/138880985/a564b132-2b76-4eb7-9603-477cc8f338e7)
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/138880985/80718e91-48be-43de-b8e2-6e27ad152130)

 I don't know how to fix it, can someone help me?

**Comment by huchenlei** [comment]
The extension does not support SDXL right now. See https://github.com/Mikubill/sd-webui-controlnet/pull/1835.
